client/components/map.jsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+
+export default class Map extends React.Component {
+  constructor(props) {
+    super(props);
+    this.mapRef = React.createRef();
+    this.mapInstance = null;
+  }
+
+  componentDidMount() {
+  }
+
+  render() {
+    return <div className="map" ref={this.mapRef} />;
+  }
+}
```

Comment:
Do you still need this?

Reply:
I saved this component for my next feature, but I'll just delete it and create a new one when I get there!